### PR TITLE
CC-2377 add remove-ip and set-ip commands

### PR DESCRIPTION
**Pull request by ssOlexBaiko**
port from https://github.com/control-center/serviced/pull/3556
[Jira](https://jira.zenoss.com/browse/CC-2377)

### CODE REVIEW

File: dao/interface.go [hunk 1]
```diff
@@ -161,6 +161,12 @@ type ControlPlane interface {
 	// Assign IP addresses to all services at and below the provided service
 	AssignIPs(assignmentRequest addressassignment.AssignmentRequest, unused *int) (err error)
 
+	// Remove the IP assignment of a service's endpoints
```
Comment (jplouis) [inline dao/interface.go]: Should not be adding new methods to the DAO package unless necessary

File: rpc/master/service_server.go [hunk 2]
```diff
@@ -152,3 +154,11 @@ func (s *Server) ClearEmergency(serviceID string, count *int) error {
 	*count = c
 	return nil
 }
+
+func (s *Server) RR(args []string) error {
```
Comment (tgjonestx) [inline rpc/master/service_server.go]: This method needs to be named `RemoveIPs`

File: rpc/master/service_client.go [hunk 3]
```diff
@@ -109,3 +110,15 @@ func (c *Client) ClearEmergency(serviceID string) (int, error) {
...
+// Remove the IP assignment of a service's endpoints
+func (c *Client) RemoveIPs(args []string) (err error) {
+	err = c.call("RemoveIPs", args, new(string))
+	return
+}
+
+// Assigns an IP address to a services that haven't IP Assignment by default
+func (c *Client) SetIPs(r addressassignment.AssignmentRequest) (err error) {
+	err = c.call("SetIPs", r, new(string))
+	return
```
Comment (jplouis) [inline rpc/master/service_client.go]: Why aren't the errors returned in these 2 methods?

File: rpc/master/service_server.go [hunk 4]
```diff
@@ -152,3 +153,13 @@ func (s *Server) ClearEmergency(serviceID string, count *int) error {
...
+
+func (s *Server) RemoveIPs(args []string, unused *string) (err error) {
+	err = s.f.RemoveIPs(s.context(), args)
+        return
+}
+
+func (s *Server) SetIPs(request addressassignment.AssignmentRequest, unused *string) (err error) {
+	err = s.f.SetIPs(s.context(), request)
+        return
+}
```
Comment (jplouis) [inline rpc/master/service_server.go]: Why aren't the errors returned in these 2 methods?

File: rpc/master/service_client.go [hunk 5]
```diff
@@ -109,3 +110,15 @@ func (c *Client) ClearEmergency(serviceID string) (int, error) {
 	err := c.call("ClearEmergency", serviceID, &affected)
 	return affected, err
 }
+
+// Remove the IP assignment of a service's endpoints
+func (c *Client) RemoveIPs(args []string) (err error) {
+	err = c.call("RemoveIPs", args, new(string))
```
Comment (jplouis) [inline rpc/master/service_client.go]: Please change these methods to "return c.call("RemoveIPs", args, new(string))" The implicit return of the error is more verbose in this case.  Same is true for the other methods doing this.

File: rpc/master/service_server.go [hunk 6]
```diff
@@ -152,3 +153,13 @@ func (s *Server) ClearEmergency(serviceID string, count *int) error {
 	*count = c
 	return nil
 }
+
+func (s *Server) RemoveIPs(args []string, unused *string) (err error) {
+	err = s.f.RemoveIPs(s.context(), args)
```
Comment (jplouis) [inline rpc/master/service_server.go]: These as well

**Pull request merged**